### ISSUE TITLE
https://gitlab.com/duplicity/duplicity

### ISSUE THREAD

**Issue by akostadinov** [henrysher/duplicity#32]
Maybe update repo description to point at the new duplicity home
https://gitlab.com/duplicity/duplicity

**Comment by joshgoebel** [comment]
Yes please, making this a pointer would be super helpful.  Seconded.
